source/lac/scalapack.cc

```diff
@@ -2617,8 +2617,8 @@ ScaLAPACKMatrix<NumberType>::save(
       chunks_size_.first  = n_rows;
       chunks_size_.second = 1;
     }
-  AssertIndexRange(chunks_size_.first + 1, n_rows);
-  AssertIndexRange(chunks_size_.second + 1, n_columns);
+  AssertIndexRange(chunks_size_.first - 1, n_rows);
```

Comment:
also the error message IMO is easier to relate for users if we write `AssertIndexRange(chunks_size_.first, n_rows+1);`